tfhe/benches/integer/signed_bench.rs

```diff
@@ -44,16 +44,45 @@ impl Default for ParamsAndNumBlocksIter {
             Err(_) => false,
         };
 
-        if is_multi_bit {
-            let params = vec![PARAM_MULTI_BIT_MESSAGE_2_CARRY_2_GROUP_2_KS_PBS.into()];
+        EnvConfig {
+            is_multi_bit,
+            is_fast_bench,
+        }
```

Comment:
we should find a way to group these things between signed and unsigned, if needed we do a separate benchmarking crate, I think this falls under the issue I tagged you on for the signed/unsigned benchmarks @soonum

Reply:
Agree, I'll to create to integrate that in `benches/utilities.rs`

Reply:
Not needed for this PR, the objective is the new benchmarks, but yeah later/follow up with the issue you were tagged in for benchmarks simplification

Reply:
Done.